Documentation/Signoffs/Sensors.md

```diff
@@ -0,0 +1,91 @@
+# Sensors System
+
+## Subsystem Function 
+
+The role of the sensor subsystem is to send signals, communicating the occupancy of the presently forefront compartment sections, to the PLC to update certain tags, so that the PLC's ladder logic code can properly use that information ad hoc.  
+
+ 
+
+Constraints 
+
+ 
+
+The Sensors: 
```

Comment:
Not specific enough for a design to be possible. Also, the constraints that are listed are ambiguous.